### PR TITLE
Upgrade mongo 3

### PR DESCRIPTION
**Pull request by aljones15**
- upgrades eslint and eslint-plugin-jsdoc
- upgrades all test dependencies to latest libraries
- lints all files with the latest linters 

What you need to do:
- [x] Check my updated JSDOC comments for inaccuracies etc.

### CODE REVIEW

File: ProfileService.js [hunk 1]
```diff
@@ -6,9 +6,9 @@
 import axios from 'axios';
 
 /**
- * @param {object} [config = {urls: {base: '/profiles'}}]
- * @param {string} [config.baseURL] - The protocol, host and port for use with
- *   node.js (e.g. https://example.com)
+ * @param {object} [config = {urls: {base: '/profiles'}}] - The config options.
+ * @param {string} [config.baseURL] - Protocol, host & port used with node.JS
```
Comment (mattcollier) [inline ProfileService.js]:
```suggestion
 * @param {string} [config.baseURL] - Protocol, host & port used with Node.js
```

File: ProfileService.js [hunk 2]
```diff
@@ -60,7 +59,7 @@ export class ProfileService {
    * @param {object} options - The options to use.
    * @param {string} [options.url = '/profiles-agents'] - The service url to
    *   use.
-   * @param {string} [options.account] - An account ID.
+   * @param {string} [options.account] - An account id.
```
Comment (mattcollier) [inline ProfileService.js]: I think in documentation `ID` is preferred.

File: ProfileService.js [hunk 3]
```diff
@@ -163,11 +159,11 @@ export class ProfileService {
...
 
   /**
-   *
    * @param {object} options - The options to use.
-   * @param {string} [options.url = '/profiles-agents'] - The service url to
+   * @param {string} [options.url='/profiles-agents'] - The service url to
    *   use.
-   *
+   * @param {string} options.account - An account id.
+   * @param {string }options.profile - A profile id.
```
Comment (mattcollier) [inline ProfileService.js]:
```suggestion
   * @param {string} options.profile - A profile id.
```

**Review by mattcollier — CHANGES_REQUESTED** [review]
Thanks for the attention to jsdocs!

**Review by dmitrizagidulin — APPROVED** [review]
👍

File: ProfileService.js (hunk 2 above)
Comment (aljones15) [reply ProfileService.js]: I checked `bedrock-account ` and saw ID over id so I made the change: https://github.com/digitalbazaar/bedrock-web-profile/pull/10/commits/45c69e5afb51e161334b90ce60249e23d2bd7db4

**Review by mattcollier — APPROVED** [review]
Thanks!

**Pull request merged**